lab10/film.py

```diff
@@ -0,0 +1,19 @@
+
+
+class Film:
+
+    def __init__(self, name, year, duration_in_minutes, genre, studio):
```

Comment:
Please add [type annotation](https://docs.python.org/3/library/typing.html) here. With the current implementation, I'm able to do something like ```my_film.name = 15``` or ```my_film.year = "some string"``` and I believe it is not meant this way.